hack/update-generated-pod-resources-dockerized.sh

```diff
@@ -23,7 +23,7 @@ set -o nounset
 set -o pipefail
 
 KUBE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd -P)"
-POD_RESOURCES_ALPHA="${KUBE_ROOT}/pkg/kubelet/apis/podresources/v1alpha1/"
+POD_RESOURCES_V1="${KUBE_ROOT}/staging/src/k8s.io/kubelet/pkg/apis/podresources/v1/"
```

Comment:
should we still generate and validate both?

Same question below for verify-generated-pod-resources.sh

Reply:
From what I've seen for other APIs we don't do that. We also don't really expect to see any changes to the beta API. It's kept for backwards compatibility and that's it :) !